File: EduHubConnect-Backend-UsingSpringBoot/src/main/java/com/connect/eduHub/dto/CommentDTO.java
package com.connect.eduHub.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.RequiredArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@RequiredArgsConstructor
public class CommentDTO {
	  private Long id;
	    private String content;
	    private Long userId;
	    private Long postId;
	    private String userName;
	    private String userProfileImage;
	    private String userJobTitle;
	    private LocalDateTime createdAt;
	    private LocalDateTime updatedAt;
}

File: EduHubConnect-AndroidFrontEnd/app/src/main/java/com/connect/eduhubconnect/dto/CommentDTO.kt
package com.connect.eduhubconnect.dto

data class CommentDTO(
    val id: Long,
    val content: String,
    val userId: Long,
    val postId: Long,
    val userName: String,
    val userProfileImage:String,
    val userJobTitle:String,
    val createdAt: String,
    val updatedAt:String
)
